### PR TITLE
Implement Chaincode dashboard

### PR DESCRIPTION
**Pull request by xichen1**
Implement chaincode dashboard. Add the required buttons to the page.
Use the dummy data for the dev purposes, will be deleted after all
features are finished.

Should be merged after #572 is merged.

Signed-off-by: xichen1 <xichen.pan@gmail.com>

### CODE REVIEW

File: src/dashboard/src/pages/ChainCode/ChainCode.js [hunk 1]
```diff
@@ -328,13 +259,43 @@ class ChainCode extends PureComponent {
       intl,
     };
 
+    const menu = record => (
+      <Menu>
+        <Menu.Item>
+          <a
+            onClick={() => {
+              this.handleDeleteChaincode(record);
```
Comment (fengyangsy) [inline src/dashboard/src/pages/ChainCode/ChainCode.js:267]: function "handleDeleteChaincode" has not defined.

File: src/dashboard/src/pages/ChainCode/ChainCode.js [hunk 2]
```diff
@@ -96,136 +89,64 @@ const UploadChainCode = props => {
...
       onOk={onSubmit}
       onCancel={() => handleModalVisible(false)}
     >
       <Form onFinish={onFinish} form={form} preserve={false}>
         <FormItem
           {...formItemLayout}
           label={intl.formatMessage({
-            id: 'app.chainCode.form.create.name',
-            defaultMessage: 'Name',
+            id: 'app.operator.chainCode.form.create.file',
```
Comment (fengyangsy) [inline src/dashboard/src/pages/ChainCode/ChainCode.js]: Can't find "app.operator.chainCode.form.create.file" in locales.

Comment (xichen1) [reply src/dashboard/src/pages/ChainCode/ChainCode.js]: it has been updated in [b2493fd](https://github.com/hyperledger/cello/pull/573/commits/b2493fdc6617449820f656ec7889edfe1dfbd7eb)

File: src/dashboard/src/pages/ChainCode/ChainCode.js [hunk 3]
```diff
@@ -335,14 +335,14 @@ class ChainCode extends PureComponent {
             <Divider type="vertical" />
             <a>
               {intl.formatMessage({
-                id: 'app.operator.chainCode.table.operate.approve',
+                id: 'app.chainCode.table.operate.approve',
```
Comment (fengyangsy) [inline src/dashboard/src/pages/ChainCode/ChainCode.js:338]: Can't find "app.chainCode.table.operate.approve" in locales

File: src/dashboard/src/pages/ChainCode/ChainCode.js [hunk 4]
```diff
@@ -335,14 +335,14 @@ class ChainCode extends PureComponent {
...
                 defaultMessage: 'Approve',
               })}
             </a>
             <Divider type="vertical" />
             <a>
               {intl.formatMessage({
-                id: 'app.operator.chainCode.table.operate.commit',
+                id: 'app.chainCode.table.operate.commit',
```
Comment (fengyangsy) [inline src/dashboard/src/pages/ChainCode/ChainCode.js:345]: Can't find "app.chainCode.table.operate.commit" in locales

File: src/dashboard/src/pages/ChainCode/ChainCode.js (hunk 3 above)
Comment (xichen1) [reply src/dashboard/src/pages/ChainCode/ChainCode.js:338]: it's added in #572, which may need to be reviewed and merged first.

I created two pr to make the purpose of each task more clear.

File: src/dashboard/src/pages/ChainCode/ChainCode.js (hunk 4 above)
Comment (xichen1) [reply src/dashboard/src/pages/ChainCode/ChainCode.js:345]: Also in #572

Comment (fengyangsy) [reply src/dashboard/src/pages/ChainCode/ChainCode.js:345]: Ok

File: src/dashboard/src/pages/ChainCode/ChainCode.js (hunk 3 above)
Comment (fengyangsy) [reply src/dashboard/src/pages/ChainCode/ChainCode.js:338]: OK

File: src/dashboard/src/pages/ChainCode/ChainCode.js (hunk 2 above)
Comment (fengyangsy) [reply src/dashboard/src/pages/ChainCode/ChainCode.js]: OK

**Review by fengyangsy — APPROVED** [review]
OK

**Pull request merged**